example/lib/slides/big_fact_slide.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_deck/flutter_deck.dart';
+
+class BigFactSlide extends FlutterDeckSlideWidget {
+  const BigFactSlide()
+      : super(
+          configuration: const FlutterDeckSlideConfiguration(
+            route: '/bigFact',
+            footer: FlutterDeckFooterConfiguration(showFooter: false),
+          ),
+        );
+
+  @override
+  FlutterDeckSlide build(BuildContext context) {
+    return FlutterDeckSlide.bigFact(
+      title: 'Let\'s Start with the decks!',
```

Comment:
Hmm, this does seem like the right usage for the template. The big fact is more about a number or short statement (title) and description below it (subtitle).

My suggestion:
Title: 100%
Subtitle: The test coverage value that flutter_deck will probably never achieve.